test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/014_UnsubscribeVehicleData_from_deprecated_fuelLevel_and_fuelLevel_State.lua

```diff
@@ -0,0 +1,34 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully process UnsubscribeVehicleData with deprecated fuelLevel and fuelLevel_State
```

Comment:
Please remote the current script and extend the checks with UnsubscribeVD in scripts with deprecated parameters(010. 011)